Header header
string type
string state
string geofence
string path